plugins/GSdx/Renderers/DX11/GSRendererDX11.cpp

```diff
@@ -26,16 +26,7 @@
 
 GSRendererDX11::GSRendererDX11()
 	: GSRendererDX(new GSTextureCache11(this), GSVector2(-0.5f))
-{
-	if (theApp.GetConfigB("UserHacks"))
-	{
-		UserHacks_unscale_pt_ln = theApp.GetConfigB("UserHacks_unscale_point_line");
-	}
-	else
-	{
-		UserHacks_unscale_pt_ln = false;
-	}
-}
+{ }
```

Comment:
I prefer
```
{
}
```
I don't think there are many that would use `{ }` with a space in between.